backend/routes/userRoutes.js

```diff
@@ -0,0 +1,58 @@
+import { Router } from 'express'
+import { db } from '../lib/db'
+import { users_db_name } from '../Utilities/API_utilities'
+
+
+const usersRouter = Router()
+
+
+usersRouter.post('/api/user', (req, res) => {
+    // It is good practice to specifically pick the fields we want to insert here *in the backend*,
+    // even if we have already done so on the front end. This is to prevent malicious users
+    // from adding unexpected fields by modifying the front end JS in the browser.
+    var newUser =  _.pick(req.body, [
+        'name', 'email', 'phone', 'passphrase'])
+    createNewUser(newUser).then(result => {
+        return res.json(result)
+    }).catch(error => {
+        console.log(error)
+        return res.status(522).json(error)
+    })
+})
+
+usersRouter.get('/api/user/:id', (req, res) => {
+    db.getById('users', req.params.id)
+    .then(user => {
+        delete user.passphrase
```

Comment:
I don't love the fact that the user object in memory has the passphrase available.  I feel like someone could very easily accidentally log the user's passphrase.  

Can we fetch all of the user fields from Mongo, minus the passphrase? 

https://stackoverflow.com/questions/12096262/how-to-protect-the-password-field-in-mongoose-mongodb-so-it-wont-return-in-a-qu

Reply:
Yes lets make an issue for this along with locking down the routes